README.md

```diff
@@ -1,3 +1,7 @@
-# AppHealthReport
+# app-health-report
 
-Module for installing AppHealthReport components
\ No newline at end of file
+This Module includes a report which collects data about failures of the any of the following components for the XXX period. 
+
+List of components: Reports, Imports, Modules, Rules, Notifications, Trackor Mail, Run Time.
+
+When report is complete successfully it is emailed to XXX address.
```

Comment:
... to the address configured in the rule.

Reply:
Done